packages/core/src/state-management/repository.ts

```diff
@@ -943,15 +949,17 @@ export class Repository {
         .then((found) => {
           const state$ = found || new RunningState(init, false)
```

Comment:
I was looking at this code for a different PR I'm working on and it made me wonder... if we hit the case here where the state isn't found in the cache or state store, but then we wind up creating a new RunningState object here, we don't ever all _registerRunningState on it, so I think it misses getting wired into the feed.  Do we need to add a call to _registerRunningState here when making a new RunningState?

Reply:
You are right, I've created #3074 to address that.